runtime/Cpp/runtime/src/atn/LexerLessAction.h

```diff
@@ -0,0 +1,57 @@
+/* Copyright (c) 2012-2017 The ANTLR Project. All rights reserved.
+ * Use of this file is governed by the BSD 3-clause license that
+ * can be found in the LICENSE.txt file in the project root.
+ */
+
+#pragma once
+
+#include "atn/LexerAction.h"
+#include "atn/LexerActionType.h"
+
+namespace antlr4 {
+namespace atn {
+
+  /// <summary>
+  /// Implements the {@code less} lexer action by calling <seealso cref="Lexer#less"/>.
+  ///
+  /// <para>The {@code less} command does not have any parameters, so this action is
+  /// implemented as a singleton instance exposed by <seealso cref="#INSTANCE"/>.</para>
+  ///
+  /// @author Sam Harwell
```

Comment:
You should actually set yourself as the author, if at all. Even if most of the code is copied. You created this file.